src/ocaml-server/main.ml

```diff
@@ -1,12 +1,21 @@
 open Lwt.Infix
 open Cohttp_lwt_unix
+open Websocket
 
-
-type flag = { modified : bool ref}
-let file_flag = {modified = ref false}
+let modified = ref true
 let visualize = ref ""
 let no_file = ref false
 let port = ref 3030
+let message = ref ""
+
+let path_to_ipm = ref ""
+exception ENV_VAR of string
+let _ = 
+  try
+    path_to_ipm := (Sys.getenv "MI_IPM")
+  with
+    _ -> raise (ENV_VAR "Environment variable MI_IPM is not set.")
```

Comment:
This still results in a fatal exception, which is not something we want the user to see:
```
Fatal error: exception Dune__exe__Main.ENV_VAR("Environment variable MI_IPM is not set.")
```

Why not something like:
```ocaml
let path_to_ipm =
  try Sys.getenv "MI_IPM"
  with Not_found -> prerr_endline "The environment variable MI_IPM must be set before running the server"; exit 1
```

You do the same thing at the bottom when checking if the specified port is in use, which I also think should be changed in a similar way.

Reply:
Replace with this, which indeed is a better option! Thanks